llvm/test/MC/Disassembler/X86/apx/reverse-encoding.txt

```diff
@@ -412,3 +412,9 @@
 # ATT:   movbeq	 %r16, %r17
 # INTEL: movbe	r17, r16
 0x62,0xec,0xfc,0x08,0x60,0xc8
+
+## cfcmov
+
+# ATT:   cfcmovbeq	%r15, %r9
```

Comment:
Need to test 16bit and 32bit too.

Reply:
Use r16 and r17 so that we can check that R2, B2 bits are decoded correctly for reverse encoding